data_steward/utils/slack_alerts.py

```diff
@@ -39,6 +42,22 @@ def _get_slack_token():
     return os.environ[SLACK_TOKEN]
 
 
+def _is_token_valid():
+    """
+    Test if the Slack token is available
+    :return:
+    """
+    client = _get_slack_client()
+    slack_token = _get_slack_token()
+    response = client.auth_test(token=slack_token)
+    if response.status_code == 200:
+        if 'ok: True' in response:
+            return True
+        else:
+            return False
+    return False
```

Comment:
We might need to add try....except for this because `client.auth_test` could throw SlackApiError. We don't want to raise SlackApiError in this case because the unit tests that import validation.main.py will fail because they don't have valid slack_token and slack_channel set up.

```suggestion
    try:
        response = client.auth_test(token=slack_token)
        return response.status_code == 200 and response['ok']
    except SlackApiError as e:
        logging.error(e)
    return False
```

Reply:
I agree that we don't need this anymore since `is_channel_available` is using the token, if the token doesn't exist, the channel won't be found right? I will remove the `is_token_valid` function.